nodes/widgets/locales/en-US/ui_table.json

```diff
@@ -8,7 +8,13 @@
             "autoCalculateColumns": "Auto Calculate Columns",
             "key": "Key",
             "label": "Label",
-            "column": "column"
+            "column": "column",
+            "selection": "Selection"
+        },
+        "selection": {
+            "none": "None",
+            "checkbox": "Multi Selection",
+            "click": "Single Row Selection"
```

Comment:
I'm not feeling these Joe. 

They are much more about "row events" than "selection"


Perhaps "Row Checkboxes" and "Row Click" are more descriptive?

![image](https://github.com/FlowFuse/node-red-dashboard/assets/44235289/7e2530fb-e927-4a43-ab90-b8c8a7d1f093)

Happy to be overridden of course.

Reply:
How about "Interaction" as the label, with "Checkboxes" and "Click" as the label options?

Reply:
Actually, I think "Selection" works better than "Interaction"